airgun/views/common.py

```diff
@@ -81,7 +84,8 @@ class listremove(SatSecondaryTab):
             TAB_NAME = 'List/Remove'
     """
     ROOT = ParametrizedLocator(
-        './/nav[@class="ng-scope"]/following-sibling::div')
+        './/nav[@class="ng-scope" or not(@class) and not(@data-block)]'
```

Comment:
anything simpler?

Reply:
i found secondary tab which has no attrs at all - just `<nav>`

Reply:
ok, spend 20 mins more on that per our discussion